src/pages/home/report/ReportActionItemDate.js

```diff
@@ -24,10 +23,6 @@ ReportActionItemDate.displayName = 'ReportActionItemDate';
 
 export default compose(
     withLocalize,
-    withOnyx({
-        currentDate: {
-            key: ONYXKEYS.CURRENT_DATE,
-        },
-    }),
```

Comment:
Hey @marcaaron I've noticed this component does not use the date received from Onyx at all 
Is this something we're passing to force a re-render?

Reply:
Hmm that's a good question. I didn't look into the original intention behind why this subscription was added. Maybe we can look at the git blame to find out more.

Reply:
Original PR is here -> https://github.com/Expensify/App/pull/3573/files

I think the intention is to force a re-render.